target/linux/ramips/mt7621/base-files/etc/board.d/01_leds

```diff
@@ -122,6 +122,11 @@ xiaomi,redmi-router-ac2100)
 youhua,wr1200js)
 	ucidef_set_led_netdev "internet" "INTERNET" "green:wan" "wan"
 	;;
+zyxel,keenetic-giga-iii)
+	ucidef_set_led_netdev  "internet" "internet"  "green:internet" "wan"
+	ucidef_set_led_netdev  "wifi5g"   "Wifi 5G"   "green:wifi5G"   "wlan0"
+	ucidef_set_led_netdev  "wifi2g"   "Wifi 2.4G" "green:wifi2G"   "wlan1"
+	;;
```

Comment:
Please use the same formatting as anybody else.

Wifi setup can be dropped here since you have DT triggers.